.github/workflows/editorconfig.yml

```diff
@@ -6,17 +6,12 @@ jobs:
     name: Editor config check
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - id: files
-      uses: jitterbit/get-changed-files@v1
-    - uses: cachix/install-nix-action@v16
-      with:
-        nix_path: nixpkgs=channel:nixos-unstable
-    - uses: workflow/nix-shell-action@v3
-      with:
-        packages: editorconfig-checker
-        script: |
-          editorconfig-checker -disable-indent-size ${{ steps.files.outputs.all }}
-    - if: ${{ failure() }}
-      run: |
-        echo "::error :: Hey! It looks like your changes don't follow our editorconfig settings."
+    - uses: actions/checkout@v3
+    - name: get changed files
+      id: changed-files
+      uses: tj-actions/changed-files@v35
+    - uses: cachix/install-nix-action@v20
+    - run: |
+      for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
```

Comment:
There seems to be a YAML syntax error here (see "actions"). Maybe this needs to be indented?